tests/integration/household_composition/test_household_question.py

```diff
@@ -0,0 +1,152 @@
+        self.assertRegex(content, 'Person Two')
+        self.assertRegex(content, 'Person Three')
+        self.assertRegex(content, 'Person Four')
+
+    def remove_answer(self, page):
+        # Add first person
+        form_data = MultiDict()
+        form_data.add("household-full-name", 'Person One')
+        form_data.add("household-full-name_1", 'Person Two')
+        form_data.add("household-full-name_2", 'Person Three')
+        form_data.add("household-full-name_3", '')
+        form_data.add("action[remove_answer]", "1") # Remove person two.
+        resp = self.client.post(page, data=form_data, follow_redirects=False)
+
+        self.assertEquals(resp.status_code, 200)
+
+        resp = self.navigate_to_page(page)
+        content = resp.get_data(True)
+        self.assertRegex(content, 'household-full-name')
+        self.assertRegex(content, 'household-full-name_2')
+        self.assertRegex(content, 'household-full-name_3')
+
+        self.assertNotRegex(content, 'household-full-name_1')
+
+    def add_answers(self):
+        self.login_and_check_introduction_text()
+        first_page = self.start_questionnaire()
+
+        # Add first person
+        form_data = MultiDict()
+        form_data.add("household-full-name", 'Person One')
+        form_data.add("action[add_answer]", "")
+        resp = self.client.post(first_page, data=form_data, follow_redirects=False)
+
+        self.assertEquals(resp.status_code, 200)
+
+        resp = self.navigate_to_page(first_page)
+        content = resp.get_data(True)
+        self.assertRegex(content, 'household-full-name_1')
+
+        # Add second person
+        form_data = MultiDict()
+        form_data.add("household-full-name", 'Person One')
+        form_data.add("household-full-name_1", 'Person Two')
+        form_data.add("action[add_answer]", "")
+        resp = self.client.post(first_page, data=form_data, follow_redirects=False)
+
+        self.assertEquals(resp.status_code, 200)
+
+        resp = self.navigate_to_page(first_page)
+        content = resp.get_data(True)
+        self.assertRegex(content, 'household-full-name_1')
+        self.assertRegex(content, 'household-full-name_2')
+
+        # Add third person
+        form_data = MultiDict()
+        form_data.add("household-full-name", 'Person One')
+        form_data.add("household-full-name_1", 'Person Two')
+        form_data.add("household-full-name_2", 'Person Three')
+        form_data.add("action[add_answer]", "")
+        resp = self.client.post(first_page, data=form_data, follow_redirects=False)
+
+        self.assertEquals(resp.status_code, 200)
+
+        resp = self.navigate_to_page(first_page)
+        content = resp.get_data(True)
+        self.assertRegex(content, 'household-full-name_1')
+        self.assertRegex(content, 'household-full-name_2')
+        self.assertRegex(content, 'household-full-name_3')
+
+        return first_page
+
+    def test_regular_submit(self):
```

Comment:
Do you mean the `test_regular_submit` test? or the `return` statement?
The test was added to cover each path through the `post_household_composition` endpoint.
Without the test the coverage drops below the acceptable level set by codecov.